.evergreen/config.yml

```diff
@@ -460,6 +473,52 @@ functions:
           MONGO_OCSP_TESTS=1 \
           .evergreen/run-ocsp-test.sh
 
+  "fetch crypt_shared":
+  - command: shell.exec
+    type: test
+    params:
+      shell: bash
+      working_dir: "src"
+      script: |
+        ${PREPARE_SHELL}
+        python3 ${DRIVERS_TOOLS}/.evergreen/mongodl.py --component=crypt_shared --version=${CRYPT_SHARED_VERSION} --out=./crypt_shared/
+        ls -R ${PROJECT_DIRECTORY}/crypt_shared
+
+  "run kmip server":
+  - command: shell.exec
+    type: test
+    params:
+      shell: bash
+      working_dir: "src"
+      background: true
+      script: |
+        ${PREPARE_SHELL}
+        cd ${DRIVERS_TOOLS}/.evergreen/csfle
+        . ./activate_venv.sh
+        # TMPDIR is required to avoid "AF_UNIX path too long" errors.
+        TMPDIR="$(dirname ${DRIVERS_TOOLS})" python3 kms_kmip_server.py
+
+  "run csfle tests":
```

Comment:
I was initially intending to have the csfle tests just run as part of the main batch, but given the combination of extra setup needed and extra restrictions on server versions, it seemed better to leave them out and run as their own task.